src/util/keyutil.ts

```diff
@@ -25,7 +25,8 @@ const KeyTypeCode = "code";
 
 type KeybindCallback = (event: WaveKeyboardEvent) => boolean;
 type KeybindConfigArray = Array<KeybindConfig>;
-type KeybindConfig = { command: string; keys: Array<string>; commandStr?: string };
+type KeybindConfig = { command: string; keys: Array<string>; commandStr?: string; info?: string };
+type KeybindUIDescription = { keys: Array<string>; info: string };
```

Comment:
add "command" into this as well.  the command name carries a lot of information and is useful even without the info sometimes.